src/workerd/io/worker.c++

```diff
@@ -1251,6 +1255,12 @@ Worker::Script::~Script() noexcept(false) {
   impl = nullptr;
 }
 
+const Worker::Isolate& Worker::Isolate::from(jsg::Lock& js) {
+  auto ptr = js.v8Isolate->GetData(3);
```

Comment:
We should define an enum somewhere of our embedder data indices so we don't accidentally reuse noe.

Reply:
Agreed. Already had a note to introduce that in a separate PR